package.json

```diff
@@ -25,6 +25,6 @@
   "license": "MIT/BSD",
   "name": "go-libp2p-gorpc",
   "releaseCmd": "git commit -a -m \"gx publish $VERSION\"",
-  "version": "1.0.24"
+  "version": "1.1.2"
```

Comment:
nicer to not jump multiple versions

Reply:
I still did a minor bump as I felt it was a significant enough change to warrant that.